features/currency/handling_disabled_currencies.feature

```diff
@@ -0,0 +1,26 @@
+@currencies
+Feature: Handling disabled currencies
+    In order to see right prices
+    As a Customer
+    I want to browse channels only with valid currency
+
+    Background:
+        Given the store operates on a channel named "Web"
+        And that channel operates on "EUR", "USD" and "GBP" currencies
+        And it uses "EUR" currency by default
+        But currency "GBP" is disabled
+
+    Scenario: Not showing the disabled currency
+        When I browse that channel
+        Then it should not have ability to switch current currency to "GBP"
+
+    Scenario: Failing to browse channel with disabled default currency
+        Given currency "EUR" is disabled as well
+        When I try to browse that channel
+        Then I should not be able to do it
+
+    Scenario: Browsing a channel with disabled default currency while using the other one
+        Given currency "EUR" is disabled as well
+        When I browse that channel while having "USD" as the current currency
+        Then it should have "USD" as the current currency
```

Comment:
According to: https://github.com/Sylius/Sylius/pull/5497/commits/b6dacb070297ccf70ec32fcd80bbfb78cb3d97c4#r70439049

`I should shop in "USD"`?